wiki/Modding_Guides/Getting_Your_Map_Modded/zh.md

```diff
@@ -39,7 +42,7 @@ Modding Queues 版块
 
 ### Pending吧
 
-传送门：http://tieba.baidu.com/f?kw=pending
+传送门：https://tieba.baidu.com/f?kw=pending
```

Comment:
【 tieba.baidu.com will redirect the https link to the http link if https is not supported for that page, and therefore the https link is still valid. 】